packages/@glimmer/runtime/test/initial-render-test.ts

```diff
@@ -46,6 +46,11 @@ class AbstractRehydrationTests extends InitialRenderSuite {
     this.assert.equal(clearedNodes.length, nodes, 'cleared nodes');
   }
 
+  assertExactServerOutput(..._expected: Content[]) {
+    let output = expect(this.serverOutput, 'must renderServerSide before calling assertServerOutput');
+    equalTokens(output, content([..._expected]));
+  }
+
   assertServerOutput(..._expected: Content[]) {
     let output = expect(this.serverOutput, 'must renderServerSide before calling assertServerOutput');
```

Comment:
I'd like to see this go through `assertExactServerOutput`

Reply:
Done.  I had honestly thought I did that the other day \:doh\: